config.yml

```diff
@@ -79,10 +78,10 @@ dind-aws:
       - docker-compose --version
 
 golang:
-  "1.11":
+  "1.13":
```

Comment:
You have to update the Dockerfile as well, can you turn it as a template like php for instance?

Reply:
yep sorry, just seen that :p

Reply:
done

Reply:
no need to maintain 1.11

Reply:
thanks, removed